<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateWin</class>
 <widget class="QWidget" name="UpdateWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>海勒ERP客户端更新窗口</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/ico/HiramClient_Icon50.ico</normaloff>:/ico/HiramClient_Icon50.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>下载状态：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labStatus">
         <property name="text">
          <string>无下载任务</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>下载速度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSpeed">
         <property name="text">
          <string>0B/S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>剩余时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelRemainTime">
         <property name="text">
          <string>0/S</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>当前下载大小：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCurrentDownload">
         <property name="text">
          <string>0B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>文件大小：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelFileSize">
         <property name="text">
          <string>0B</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRetry">
       <property name="text">
        <string>重新下载</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
